src/task/create-item-validation-task.test.ts

```diff
@@ -0,0 +1,98 @@
+  s3: {
+    s3Region: 's3Region',
+    s3Bucket: 's3Bucket',
+    s3AccessKeyId: 's3AccessKeyId',
+    s3SecretAccessKey: 's3SecretAccessKey',
+  },
+  local: {
+    storageRootPath: 'storageRootPath',
+  },
+}, 0);
+const runner = new Runner();
+
+describe('Run detect bad words process', () => {
+  const iVP = [SAMPLE_VALIDATION_PROCESS[0]];
+  const iVId = 'item-validation-id-1';
+
+  it('Detect fields containing bad words', async () => {
+    const input = BAD_ITEM.id;
+    const item = BAD_ITEM;
+    const task = new CreateItemValidationTask(member, validationService, itemService, fTM, runner, ITEM_TYPE.LOCALFILE, { itemId: input });
+    jest.spyOn(itemService, 'get').mockImplementation(async () => item as Item);
+    jest.spyOn(validationService, 'getEnabledProcesses').mockImplementation(async () => iVP);
+    jest
+      .spyOn(validationService, 'getItemValidationStatuses')
+      .mockImplementation(async () => IVStauses);
+    jest
+      .spyOn(validationService, 'getItemValidationReviewStatuses')
+      .mockImplementation(async () => IVStauses);
+    jest
+      .spyOn(validationService, 'createItemValidation')
+      .mockImplementation(async () => iVId);
+    jest
+      .spyOn(validationService, 'createItemValidationGroup')
+      .mockImplementation(async () => itemValidationGroupEntry);
+    jest
+      .spyOn(validationService, 'updateItemValidationGroup')
+      .mockImplementation(async () => itemValidationGroupEntry);
+    jest
+      .spyOn(validationService, 'createItemValidationReview')
+      .mockImplementation(async () => ITEM_VALIDATION_REVIEWS[0] as ItemValidationReview);
+    await task.run(handler);
+    expect(task.result).toEqual('pending manual review');
+  });
+
+  it('Return empty list if item is OK', async () => {
+    const input = GOOD_ITEM.id;
+    const item = GOOD_ITEM;
+    const task = new CreateItemValidationTask(member, validationService, itemService, fTM, runner, ITEM_TYPE.LOCALFILE, { itemId: input });
+    jest.spyOn(itemService, 'get').mockImplementation(async () => item as Item);
+    jest.spyOn(validationService, 'getEnabledProcesses').mockImplementation(async () => iVP);
+    jest
+      .spyOn(validationService, 'createItemValidation')
+      .mockImplementation(async () => iVId);
+    jest
+      .spyOn(validationService, 'getItemValidationStatuses')
+      .mockImplementation(async () => IVStauses);
+    jest
+      .spyOn(validationService, 'getItemValidationReviewStatuses')
+      .mockImplementation(async () => IVStauses);
+    jest
+      .spyOn(validationService, 'createItemValidationGroup')
+      .mockImplementation(async () => itemValidationGroupEntry);
+    jest
+      .spyOn(validationService, 'updateItemValidationGroup')
+      .mockImplementation(async () => itemValidationGroupEntry);
+    jest
+      .spyOn(validationService, 'createItemValidationReview')
+      .mockImplementation(async () => ITEM_VALIDATION_REVIEWS[0] as ItemValidationReview);
```

Comment:
Can you factor out so you don't copy/paste this code twice? See https://github.com/graasp/graasp-plugin-thumbnails/blob/main/test/mock.ts